ROADMAP.md

```diff
@@ -9,46 +9,38 @@ Any dates listed below and the specific issues that will ship in a given milesto
 See the [Github project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
 
-## Rook 1.6
+## Rook 1.7
 
-The following high level features are targeted for Rook v1.6 (March 2021). For more detailed project tracking see the [v1.6 board](https://github.com/rook/rook/projects/20).
+The following high level features are targeted for Rook v1.7 (July 2021). For more detailed project tracking see the [v1.7 board](https://github.com/rook/rook/projects/21).
 
 * Ceph
-  * Support the Ceph Pacific release
-  * CephFS Mirroring support for data replication across clusters
-  * RBD mirroring configuration between clusters to support application DR (disaster recovery)
+  * Helm chart for the cluster CR [#2109](https://github.com/rook/rook/issues/2109)
   * RGW Multi-site replication improvements towards declaring the feature stable
-  * Upgrade OSDs for large clusters in parallel within failure domains
-  * Configure bucket notifications with a CRD [#6425](https://github.com/rook/rook/issues/6425)
-* Storage Providers
-  * Clearly define maintainer expectations for storage providers
-  * Remove unmaintained storage providers: EdgeFS, CockroachDB, and YugabyteDB
+  * Configure bucket notifications with a CRD ([design doc](https://github.com/rook/rook/blob/master/design/ceph/object/ceph-bucket-notification-crd.md))
```

Comment:
@travisn CSI windows support will be part of 1.7 or later versions?

Reply:
@Madhu-1 What is the timeline for the windows node support from the CSI driver? I'm assuming it's too early to commit to v1.7, so I'll add it to the future work section for now.

Reply:
Yes, it would be too early. we are planning for 3.4 of cephcsi not sure it's going to be ready. good to add it to the future section :+1: